app/models/customization.rb

```diff
@@ -0,0 +1,11 @@
+# typed: strict
+class Customization < ApplicationRecord
+  belongs_to :item
+
+  # Define the attributes with their respective data types and constraints
+  attribute :id, :integer
+  attribute :created_at, :datetime
+  attribute :updated_at, :datetime
+  attribute :customization_details, :text
+  attribute :item_id, :integer
+end
```

Comment:
The Customization model defines attributes explicitly, which is redundant since Rails automatically defines these based on the database schema. You can remove the explicit attribute definitions to clean up the model file.